xbmc/music/MusicDatabase.cpp

```diff
@@ -3946,11 +3946,38 @@ void CMusicDatabase::UpdateTables(int version)
       m_pDS->exec("DROP TABLE album");
       m_pDS->exec("ALTER TABLE album_new RENAME TO album");
    }
+  if (version < 55)
+  {
+      // Convert the content table from using musicdb:// paths to actual paths
+      if (NULL == m_pDB.get()) return;
+      if (NULL == m_pDS.get()) return;
+      //Genres are no longer supported here
+      m_pDS->exec(PrepareSQL("DELETE FROM content WHERE strPath LIKE 'musicdb://genres/%'"));
+
+      m_pDS->query(PrepareSQL("SELECT strPath FROM content"));
+
+      std::vector<std::string> items;
+      // get data from returned rows
+      while (!m_pDS->eof())
+      {
+        items.push_back(m_pDS->get_sql_record()->at(0).get_asString());
```

Comment:
No specific reason, had some problems getting this to work, might have just been the first thing that woked. Will change.